repository/d.json

```
@@ -508,6 +508,16 @@
 				}
 			]
 		},
+		{
+			"name": "Definition Preview",
+			"details": "https://github.com/nickav/DefinitionPreview",
+			"releases": [
+				{
+					"sublime_text": "*",
```

AFAICS you plugin is ST4 only

```suggestion
					"sublime_text": ">=4000",
```